File: src/main/java/kr/co/lunatalk/domain/category/dto/request/CategoryAddProductRequest.java
package kr.co.lunatalk.domain.category.dto.request;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotNull;

import java.util.List;

public record CategoryAddProductRequest(
	@NotNull(message = "상품을 선택해주세요.")
	@Schema(description = "추가할 상품 Ids")
	List<Long> productIds
	) {
}

File: src/main/kotlin/kr/co/lunatalk/domain/category/dto/request/CategoryAddProductRequest.kt
package kr.co.lunatalk.domain.category.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotNull

data class CategoryAddProductRequest(
    @field:NotNull(message = "상품을 선택해주세요.")
    @Schema(description = "추가할 상품 Ids")
    val productIds: List<Long>
)
